### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ymalen**
Update the documentation to note an issue with using stream and the api client.

### CODE REVIEW

**Comment by yliaog** [comment]
please sign cla

File: README.md [hunk 1]
```diff
@@ -185,3 +185,8 @@ Specifically check `ipaddress` and `urllib3` package versions to make sure they
 Starting from 4.0 release, we do not support directly calling exec or attach calls. you should use stream module to call them. so instead
 of `resp = api.connect_get_namespaced_pod_exec(name, ...` you should call `resp = stream(api.connect_get_namespaced_pod_exec, name, ...`.
 See more at [exec example](examples/exec.py).
+
+    Using Stream will overwrite the requests protocol in _core_v1_api.CoreV1Api()_
```
Comment (roycaihw) [inline README.md]: please remove the indention to fix the rendering

File: README.md [hunk 2]
```diff
@@ -185,3 +185,8 @@ Specifically check `ipaddress` and `urllib3` package versions to make sure they
 Starting from 4.0 release, we do not support directly calling exec or attach calls. you should use stream module to call them. so instead
 of `resp = api.connect_get_namespaced_pod_exec(name, ...` you should call `resp = stream(api.connect_get_namespaced_pod_exec, name, ...`.
 See more at [exec example](examples/exec.py).
+
+    Using Stream will overwrite the requests protocol in _core_v1_api.CoreV1Api()_ 
+    This will cause a failure in  non-exec/attach calls. If you reuse your api client object, you will need to 
+    recreate it between api calls that use _stream_ and other api calls.
```
Comment (roycaihw) [inline README.md]: the above means `api` needs to get recreated after the exec calls (if there are more non-exec/attach calls)? 
https://github.com/kubernetes-client/python/blob/da8c843bda7ebf03f53cdc95409e5543d2b2abd3/examples/exec.py#L13

**Comment by scottilee** [comment]
@ymalen any update on this?

File: README.md (hunk 2 above)
Comment (ymalen) [reply README.md]: That is correct.

**Comment by ymalen** [comment]
I signed the CLA

**Pull request merged**